packages/widget/src/controllers/transfers/evm/execute.ts

```diff
@@ -19,6 +22,7 @@ export async function executeNextEvmTransaction(
   if (this.getTransferState() === FungibleTransferState.PENDING_APPROVALS) {
     this.waitingUserConfirmation = true;
     this.host.requestUpdate();
+    const transactions = [];
```

Comment:
it'd be better to put type assertion here because it currently resolves to `any[]`
```suggestion
    const transactions: UnsignedTransaction[] = [];
```